buildspec.yml

```diff
@@ -4,13 +4,13 @@ phases:
   pre_build:
     commands:
       - echo Build started on `date`
-      - mvn install:install-file -Dfile=./src/main/resources/ojdbc7-12.1.0.2.jar -DgroupId=com.oracle -DartifactId=ojdbc7 -Dversion=12.1.0.2 -Dpackaging=jar
+      - mvn install
```

Comment:
no longer using oracleDB, so installing specific file "ojdbc7-12-1-0.2.jar" will not be needed to be installed.